platforms/Windows/runtime-amd64.wxs

```diff
@@ -21,7 +21,7 @@
       </Directory>
     </Directory>
 
-    <SetDirectory Id="INSTALLDIR" Value="[ProgramFiles64Folder]swift">
+    <SetDirectory Id="INSTALLDIR" Value="[ProgramFiles64Folder]Swift">
```

Comment:
This was intentionally `swift`.  I don't think that there is any strict suggestion/requirement/pattern that says that this should be `Swift`.

Reply:
MSDN suggests the installation path to be `[Program64FilesFolder]\ISV Name\Application Name\x64` (take x86_64 as example), and it’s clear to use title case with respect to blank spaces.

In practice, `ISV Name` is sometimes ignored, arch directory is ignored if it doesn’t support multi-arch installation, major version number is appended after `Application Name` if it supports multi-version installation.

This means if possible we should use `[Program64FilesFolder]\Swift\5.8-dev\runtime\x64` or `[Program64FilesFolder]\Swift\5.8-dev\runtime`.

Reply:
On my machine this rule gets fully addressed except for `dotnet` and `nodejs`. I don’t think we should be the rule breaker if we can avoid that.

<img width="456" alt="image" src="https://user-images.githubusercontent.com/26525394/185778150-a05418a5-8845-439d-bd60-36662999851d.png">

Reply:
"ISV Name" uses title case, and I believe you’re confusing the "Application Name" here. An application on Windows can contain several components.

For Swift, "ISV Name" should be `Swift` or `Swift.org`, and "Application Name" should be `Swift`.

Reply:
`Swift` looks more fit and I don’t think we have reason to refuse that🤷‍♂️ Other OSS projects like Git, CMake and Vim also respects title case on Windows — when in Rome, do as Romans do.

Also, according to Swift editorial guideline:
> When using the name Swift in titles, headlines, or body copy, always typeset Swift with an uppercase S followed by lowercase letters.

We should prefer `Swift` to `swift` where title case is allowed.